lib/hardware_services/network_drivers/hilapi.py

```diff
@@ -0,0 +1,107 @@
+# this is a library to conveniently do REST calls to HIL server
+
+import requests
+import sys
+import yaml
+
+def error_check(response):
```

Comment:
Didn't we want to make the inventory and network layers properly python classes instead of using a module level implementation?

I would be open to merging this as is, but then apply a PR that refactors them properly.

Or, if you'd rather, let's do the refactor as part of this commit?